src/components/views/dialogs/MultiInviteDialog.js

```diff
@@ -41,20 +42,28 @@ export default class MultiInviteDialog extends React.Component {
         }
     }
 
+    componentWillUnmount() {
+        this._unmounted = true;
```

Comment:
should we also be setting `_canceled` here? What happens if the user closes the dialog without clicking 'cancel'? I'm a bit confused about the difference between `_unmounted` and `_canceled`, and why we handle them differently in `_inviteMore`. Perhaps some comments would help?